WasatchNET/EEPROM.cs

```diff
@@ -795,9 +754,73 @@ public string productConfiguration
         }
 
         string _productConfiguration;
+
+        public PAGE_SUBFORMAT subformat
+        {
+            get { return _subformat; }
+            set
+            {
+                EventHandler handler = EEPROMChanged;
+                _subformat = value;
+                handler?.Invoke(this, new EventArgs());
+            }
+        }
+
+        PAGE_SUBFORMAT _subformat;
+
         /////////////////////////////////////////////////////////////////////////       
-        // Pages 6-7 unallocated
+        // Page 6
+        /////////////////////////////////////////////////////////////////////////  
+        public float[] intensityCorrectionCoeffs
+        {
+            get { return _intensityCorrectionCoeffs; }
+            set
+            {
+                EventHandler handler = EEPROMChanged;
+                _intensityCorrectionCoeffs = value;
+                handler?.Invoke(this, new EventArgs());
+            }
+        }
+
+        float[] _intensityCorrectionCoeffs;
+
+        public byte intensityCorrectionOrder
+        {
+            get { return _intensityCorrectionOrder; }
+            set
+            {
+                EventHandler handler = EEPROMChanged;
+                _intensityCorrectionOrder = value;
+                handler?.Invoke(this, new EventArgs());
+            }
+        }
+
+        byte _intensityCorrectionOrder;
+
         /////////////////////////////////////////////////////////////////////////       
+        // Page 7 unallocated
+        /////////////////////////////////////////////////////////////////////////  
+
+        /////////////////////////////////////////////////////////////////////////
+        // Compound Fields
+        /////////////////////////////////////////////////////////////////////////
+
+        public string fullModel
+        {
+            get { return _model + _productConfiguration; }
+            set
+            {
+                if (value.Length > 16)
+                {
+                    _model = value.Substring(0, 16);

```

Comment:
I haven't fully looked through this, but by writing _model rather than model, aren't you skipping the setters for model and productConfiguration (including the EEPROMChanged event handler)?

Reply:
This is true, might as well go through the setters